fwoa-tools/src/exportHelper.ts

```diff
@@ -5,159 +5,126 @@
-      const statusPollUrl = response.headers['content-location'];
-      console.log('Beginning export, check status with: ', statusPollUrl);
-      return statusPollUrl;
-    } catch (e) {
-      console.error('Failed to start export job', e);
-      throw e;
+export async function startExportJob(startExportJobParam: StartExportJobParam): Promise<GlueJobResponse> {
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const params: any = {
+      _outputFormat: 'ndjson'
+    };
+    if (startExportJobParam.since) {
+      params._since = startExportJobParam.since;
     }
+    const glue = new Glue({
+      region: process.env.API_AWS_REGION
+    });
+    const jobId = uuidv4();
+    const startJobRunResponse = await glue
+      .startJobRun({
+        JobName: startExportJobParam.glueJobName,
+        Arguments: {
+          '--jobId': jobId,
+          '--jobOwnerId': 'FWoAMigrationClient',
+          '--exportType': 'system',
+          '--transactionTime': new Date().toISOString(),
+          '--since': startExportJobParam.since || '1800-01-01T00:00:00.000Z',
+          '--type': EXPORT_TYPES,
+          '--outputFormat': 'ndjson',
+          '--tenantId': startExportJobParam.tenantId!,
+          '--serverUrl': startExportJobParam.apiUrl,
+          '--snapshotExists': startExportJobParam.snapshotExists.toString().toLowerCase(),
+          '--snapshotLocation': startExportJobParam.snapshotLocation
+        }
+      })
+      .promise();
+    const jobRunId = startJobRunResponse.JobRunId!;
+    return {
+      jobId,
+      jobRunId
+    };
+  } catch (e) {
+    console.error('Failed to start export job', e);
+    throw e;
   }
+}
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public async getExportStatus(statusPollUrl: string): Promise<any> {
-    const cutOffTime = new Date(new Date().getTime() + MAX_EXPORT_RUNTIME);
-    while (new Date().getTime() < cutOffTime.getTime()) {
-      try {
-        // eslint-disable-next-line no-await-in-loop
-        const response = await this.fhirUserAxios.get(statusPollUrl);
-        if (response.status === 200) {
-          return response.data;
-        }
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export async function getExportStatus(jobName: string, jobId: string, jobRunId: string): Promise<any> {
+  const cutOffTime = new Date(new Date().getTime() + MAX_EXPORT_RUNTIME);
+  const glue = new Glue({
+    region: process.env.API_AWS_REGION
+  });
+  while (new Date().getTime() < cutOffTime.getTime()) {
+    try {
+      // eslint-disable-next-line no-await-in-loop
+      const jobRun = await glue
+        .getJobRun({
+          JobName: jobName,
+          RunId: jobRunId
+        })
+        .promise();
+      const state = jobRun.JobRun!.JobRunState!;
+      if (state === 'SUCCEEDED') {
+        return state;
+      } else if (state === 'RUNNING' || state === 'WAITING') {
         // eslint-disable-next-line no-await-in-loop
```

Comment:
We should probably also add a log line to let user know that we're still waiting. Otherwise users won't know if the script is still running as expected. Maybe something like, `Glue job is still in ${state} state`